pkg/kubectl/history.go

```diff
@@ -111,7 +113,12 @@ func (h *DeploymentHistoryViewer) ViewHistory(namespace, name string, revision i
 		if !ok {
 			return "", fmt.Errorf("unable to find the specified revision")
 		}
-		return printTemplate(template)
+
+		templateString, err := printTemplate(template)
+		if err != nil {
+			return "", err
+		}
+		return "Revision Time:\t" + revisionTime[revision] + "\n" + templateString, nil
```

Comment:
For the new replicaset object, there should be annotations already. But for compatibility issues, I think it's ok to add protection in case there are old object in the cluster. Thanks for the comments, I'll add a protection and update the patch.